OptionSelector/OptionsTableViewController.m

```diff
@@ -0,0 +1,110 @@
+//
+//  OptionsTableViewController.m
+//  OptionSelector
+//
+//  Created by Zoufishan Mehdi on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OptionsTableViewController.h"
+
+@interface OptionsTableViewController ()
+
+@end
+
+@implementation OptionsTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+}
+
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    // Return the number of sections.
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    // Return the number of rows in the section.
+    return self.category.options.count;
+}
+
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"OptionName" forIndexPath:indexPath];
+    
+    cell.textLabel.text = [self.category.options objectAtIndex:indexPath.row];
+    
+    //set checkmark methods
+  cell.accessoryType = UITableViewCellAccessoryNone;
```

Comment:
This line should be indented two more spaces.

(I'm a bit of a stickler for indentation; bad indentation makes code hard to read and can mask bugs.)